src/content/app/genome-browser/hooks/useGenomeBrowserAnalytics.ts

```diff
@@ -0,0 +1,303 @@
+      app: AppName.GENOME_BROWSER
+    });
+  };
+
+  const trackTrackSettingsOpened = (trackId: string) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: trackId,
+      action: 'opened'
+    });
+  };
+
+  const trackRegionChange = () => {
+    sendTrackEvent({
+      category: 'browser_navigation',
+      label: 'region_editor',
+      action: 'change_region'
+    });
+  };
+
+  const trackRegionFieldChange = () => {
+    sendTrackEvent({
+      category: 'browser_navigation',
+      label: 'region_field',
+      action: 'change_region'
+    });
+  };
+
+  const trackBookmarksDrawerOpened = (totalBookmarks: number) => {
+    sendTrackEvent({
+      category: 'drawer_open',
+      label: 'recent_bookmarks',
+      action: 'clicked',
+      value: totalBookmarks
+    });
+  };
+
+  const trackBookmarksLinkClicked = (
+    objectType: string,
+    linkPositionFromTop: number
+  ) => {
+    sendTrackEvent({
+      category: 'recent_bookmark_link',
+      label: objectType,
+      action: 'clicked',
+      value: linkPositionFromTop
+    });
+  };
+
+  const trackTrackNameToggle = (
+    selectedCog: string,
+    isTrackNameShown: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'track_name_' + (isTrackNameShown ? 'on' : 'off')
+    });
+  };
+
+  const trackFeatureLabelToggle = (
+    selectedCog: string,
+    areFeatureLabelsShown: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'feature_labels_' + (areFeatureLabelsShown ? 'on' : 'off')
+    });
+  };
+
+  const trackShowSeveralTranscriptsToggle = (
+    selectedCog: string,
+    isSeveralTranscriptsShown: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action:
+        'several_transcripts_' + (isSeveralTranscriptsShown ? 'on' : 'off')
+    });
+  };
+
+  const trackShowTranscriptsIdToggle = (
+    selectedCog: string,
+    shouldShowTranscriptIds: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'transcript_labels_' + (shouldShowTranscriptIds ? 'on' : 'off')
+    });
+  };
+
+  const trackApplyToAllInTackSettings = (
+    selectedCog: string,
+    shouldApplyToAll: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'apply_to_all - ' + (shouldApplyToAll ? 'unselected' : 'selected')
+    });
+  };
+
+  const trackTrackPanelTabChange = (selectedTrackPanelTab: TrackSet) => {
+    sendTrackEvent({
+      category: 'track_panel_tab',
+      label: selectedTrackPanelTab,
+      action: 'selected'
+    });
+  };
+
+  const trackTrackPanelSectionToggled = (sectionName: string) => {
+    sendTrackEvent({
+      category: 'track_panel',
+      label: sectionName,
+      action: 'section_toggled'
+    });
+  };
+
+  const trackResetIcon = () => {
```

Comment:
`trackResetIconClick`? Or perhaps `trackFocusObjectReset`?

Reply:
Changed it to `trackFocusObjectReset`